lib/smart_answer_flows/simplified-expenses-checker.rb

```diff
@@ -70,31 +70,49 @@ def define
       #
       # if no go to Q4
       multiple_choice :capital_allowances? do
-        option :yes
+        option :capital_allowance
+        option :simplified_expenses
         option :no
 
         next_node do |response|
-          calculator.capital_allowance = response
-
-          if response == "yes"
-            if calculator.any_work_location?
-              if calculator.working_from_home?
-                # Q11
-                question :hours_work_home?
-              else
-                # Q13
-                question :deduct_from_premises?
-              end
-            else
+          calculator.selected_allowance = response
+          case
+          when calculator.capital_allowance?
+            if calculator.vehicle_only?
               outcome :capital_allowance_result
+            elsif calculator.home?
+              question :hours_work_home?
+            elsif calculator.business_premises?
+              question :deduct_from_premises?
+            end
+          when calculator.simplified_expenses?
+            if calculator.vehicle_only?
+              outcome :you_cant_claim_capital_allowance
+            elsif calculator.home?
+              question :hours_work_home?
+            elsif calculator.business_premises?
+              question :deduct_from_premises?
+            end
+          when calculator.no_allowance?
+            if calculator.van? || calculator.motorbike?
+              question :how_much_expect_to_claim?
+            elsif calculator.car?
+              question :car_status_before_usage?
             end
-          else
-            question :how_much_expect_to_claim?
           end
         end
       end
 
       #Q5 - Was your car new or second-hand when you started using it for your business?
+      multiple_choice :car_status_before_usage? do
```

Comment:
Thanks. That's makes the new question easier to understand. Could you update the commit message to reflect that you are adding a new question?